src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -390,26 +390,6 @@ def test_remove_daemon(self, cephadm_module):
             out = wait(cephadm_module, c)
             assert out == ["Removed rgw.myrgw.myhost.myid from host 'test'"]
 
-    @mock.patch("cephadm.module.CephadmOrchestrator._run_cephadm", _run_cephadm(
```

Comment:
Care to re-add a unit test for this?

Reply:
I did add this in https://github.com/ceph/ceph/pull/34617 but that would require some more cherry-picking. Sigh.

Reply:
ok, nvm

Reply:
just added those commits to this PR.